core/server/worker/src/test/java/alluxio/worker/block/allocator/AllocatorTestBase.java

```diff
@@ -74,6 +74,11 @@ public void before() throws Exception {
     resetManagerView();
   }
 
+  /**
+   * Resets the manager view configured by the parameters in this base class.
+   *
+   * @throws Exception when error happens during creating temporary folder
```

Comment:
These tests are never supposed to throw exceptions, so there's no need to include `@throws` javadoc